wd_util.c

```diff
@@ -91,6 +91,7 @@ int wd_init_ctx_config(struct wd_ctx_config_internal *in,
 	}
```

Comment:
if wd_init_ctx_config is used by all algorithms, we can simply add pid check inside wd_init_ctx_config instead of every algorithms.c
wd_init_ctx_config
   if (in->pid == getpid)
    return;

Reply:
Yes，this way is better.